external/fv3fit/fv3fit/keras/_models/precipitative.py

```diff
@@ -134,6 +135,9 @@ class PrecipitativeHyperparameters(Hyperparameters):
     loss: LossConfig = LossConfig(scaling="standard", loss_type="mse")
     couple_precip_to_dQ1_dQ2: bool = True
     normalization_fit_samples: int = 500_000
+    callback_config: CallbackConfig = dataclasses.field(
```

Comment:
suggestion: you can avoid a yaml attribute with only one attribute by putting the list of individual callback configs here, instead of packaging them inside a dataclass.